src/supported-modules.h

```diff
@@ -129,6 +129,7 @@ static pam_module_t *module_list_auth[] = {
   &mod_pam_fprint,
   &mod_pam_fprintd,
   &mod_pam_thinkfinger,
+  &mod_pam_himmelblau,
   &mod_pam_gnome_keyring,
```

Comment:
FYI, `pam_himmelblau` also must come before `pam_gnome_keyring` in the auth stack, since `pam_gnome_keyring` sends a password prompt (which isn't required for the Himmelblau passwordless MFA and Hello flows).

Reply:
`pam_gnome_keyring` is asking for a password to unlock the keyring. But since I don't use GNOME I cannot say if this breaks something or not.

Reply:
Yes, I know it breaks keyring unlock. I have a workaround which can be implemented once they have Hello PINs configured.